translations/de.po

```diff
@@ -337,13 +348,13 @@ msgid "Apply the matrix"
 msgstr "Die Matrix anwenden"
 
 msgid "Other"
-msgstr ""
+msgstr "Anderes"
 
 msgid "Optimize"
-msgstr ""
+msgstr "Optimieren"
 
 msgid "Any changes will apply immediately."
-msgstr ""
+msgstr "Änderungen werden gleich eingesetzt."
```

Comment:
I'd change this to "Änderungen werden sofort übernommen." or "Änderungen werden sofort wirksam.".
"gleich" can mean _right now_, but also _a short amount of time later_ depending on the context. In this case, it's not fully clear.